packages/api/tests/endpoints/test-granules.js

```diff
@@ -25,21 +36,44 @@ const {
 const launchpad = require('@cumulus/launchpad-auth');
 const { randomString, randomId } = require('@cumulus/common/test-utils');
 const { getBucketsConfigKey, getDistributionBucketMapKey } = require('@cumulus/common/stack');
+
+// PG mock data factories
+const {
+  fakeGranuleRecordFactory,
+  fakeCollectionRecordFactory,
+} = require('@cumulus/db/dist/test-utils');
+
 const assertions = require('../../lib/assertions');
 const models = require('../../models');
 const bootstrap = require('../../lambdas/bootstrap');
 const indexer = require('../../es/indexer');
+
+// Dynamo mock data factories
 const {
   fakeAccessTokenFactory,
   fakeCollectionFactory,
   fakeGranuleFactoryV2,
   createFakeJwtAuthToken,
   setAuthorizedOAuthUsers,
 } = require('../../lib/testUtils');
+
 const {
   createJwtToken,
 } = require('../../lib/token');
 const { Search } = require('../../es/search');
+const { migrationDir } = require('../../../../lambdas/db-migration');
+
+const testDbName = `granules_${cryptoRandomString({ length: 10 })}`;
+
+let esClient;
```

Comment:
Nit: alpha sort

Reply:
(these truly are nits/style concerns, only noting because I went into a few of them looking at the review and reflexively sorted locally).     There's room to discuss it as a style nit/concern elsewhere and shouldn't hold up the PR

Reply:
👍 yup! No worries! I just need to get in the habit.

19d1a6cb4175af68b8e1ddc1d7c27a30ea3ebdc7